recipes/r-qgisprocess/meta.yaml

```diff
@@ -0,0 +1,106 @@
+    (plugin) processing providers are supported.  Beside referring data sources from
+    file, also common objects from 'sf', 'terra' and 'stars' are supported. The native
+    processing algorithms are documented by QGIS.org (2023) <https://docs.qgis.org/latest/en/docs/user_manual/processing_algs/>.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - thewchan
+
+# The original CRAN metadata for this package was:
+
+# Package: qgisprocess
+# Title: Use 'QGIS' Processing Algorithms
+# Version: 0.1.0
+# Authors@R: c( person("Dewey", "Dunnington", , "dewey@fishandwhistle.net", role = "aut", comment = c(ORCID = "0000-0002-9415-4582", affiliation = "Voltron Data")), person("Floris", "Vanderhaeghe", , "floris.vanderhaeghe@inbo.be", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-6378-6229", affiliation = "Research Institute for Nature and Forest (INBO)")), person("Jan", "Caha", , "jan.caha@outlook.com", role = "aut", comment = c(ORCID = "0000-0003-0165-0606")), person("Jannes", "Muenchow", , "malnamalja@gmx.de", role = "aut", comment = c(ORCID = "0000-0001-7834-4717")), person("Antony", "Barja", , "antony.barja8@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5921-2858")), person("Robin", "Lovelace", , "rob00x@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5679-6536")), person("Jakub", "Nowosad", , "nowosad.jakub@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-1057-3721")), person("Research Institute for Nature and Forest (INBO)", role = c("cph", "fnd"), comment = "https://www.inbo.be/en/") )
+# Description: Provides seamless access to the 'QGIS' (<https://qgis.org/en/site/>) processing toolbox using the standalone 'qgis_process' command-line utility.  Both native and third-party (plugin) processing providers are supported.  Beside referring data sources from file, also common objects from 'sf', 'terra' and 'stars' are supported. The native processing algorithms are documented by QGIS.org (2023) <https://docs.qgis.org/latest/en/docs/user_manual/processing_algs/>.
+# License: GPL (>= 3)
+# URL: https://r-spatial.github.io/qgisprocess/, https://github.com/r-spatial/qgisprocess
+# BugReports: https://github.com/r-spatial/qgisprocess/issues
+# Depends: R (>= 3.6.0)
+# Imports: assertthat, glue, jsonlite, processx (>= 3.5.2), rappdirs, rlang, stats, stringr, tibble, vctrs, withr
+# Suggests: dplyr, knitr, mapview, raster, rmarkdown, rprojroot, sf, spDataLarge, stars, stringi, terra, testthat, tidyr
+# VignetteBuilder: knitr
+# Additional_repositories: https://geocompr.r-universe.dev
+# Config/checklist/communities: inbo
+# Config/checklist/keywords: R; package; QGIS
+# Encoding: UTF-8
+# Language: en-GB
+# RoxygenNote: 7.2.3
+# SystemRequirements: 'QGIS' latest or long-term release (>= 3.28). Older 'QGIS' releases are not officially supported, but may work since 'QGIS' 3.16.
```

Comment:
The recipe appears not to address this. This should be added as `run:` requirement.

One can also see it failing in the CI test (though it exits with 0):

<details>

```r
R version 4.2.3 (2023-03-15) -- "Shortstop Beagle"
Copyright (C) 2023 The R Foundation for Statistical Computing
Platform: x86_64-conda-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> library('qgisprocess')
Attempting to load the package cache ... No cache found.
Will try to reconfigure qgisprocess and build new cache ...

>>> PROBLEM encountered: couldn't build package cache! <<<
The 'qgis_process' command-line utility was either not found or
did not fulfil the needs to build the package cache.
Please run `qgis_configure()` to fix this and rebuild the cache.
See its documentation if you need to preset the path of qgis_process.
If the problem persists, make sure that you correctly installed QGIS
for your operating system using the instructions at
https://download.qgis.org./
> 
>
```

</details>

I would recommend that some integration test be run - something that will actually fail if the QGIS is not usable. Does this package has `testthat` tests that could be run?

Reply:
I'll add qgis first; I'll look to see if they have tests in place